pkg/assets/create/creater.go

```diff
@@ -57,45 +55,50 @@ func EnsureManifestsCreated(ctx context.Context, manifestDir string, restConfig
 		options.StdErr = os.Stderr
 	}
 
-	// Default QPS in client (when not specified) is 5 requests/per second
-	// This specifies the interval between "create-all-resources", no need to make this configurable.
-	interval := 200 * time.Millisecond
```

Comment:
This interval just match the client QPS, to the `PollImmediateUntil(200*time.Millisecond)` just does what client does (there is no delay).